app.js

```diff
@@ -34,3 +34,52 @@ var stuff = [
 //   // console.log(stuff[i].name);
 //   // console.log(stuff[i].src);
 // }
+
+function findRandom(){
+  var tempArray = [];
+  while(tempArray.length < 3){
+    var rand = stuff[Math.floor(Math.random() * stuff.length)].src;
+    if(tempArray.indexOf(rand) === -1){
+      tempArray.push(rand);
+    }
+    // console.log(tempArray);
+  }
+  randomNumArray = tempArray;
+}
+findRandom();
```

Comment:
Group your function calls together at the bottom instead of interspersing them throughout the file. As-is, it's really hard to figure out where you call this function.